rules/linux/execution_vi_binary.toml

```
@@ -25,8 +26,8 @@ type = "eql"
 
 query = '''
 sequence by host.id,process.pid with maxspan=1m
-[process where process.name == "vi" and process.args : "-c" and process.args : (":!/bin/bash", ":!/bin/sh", ":!bash", ":!sh")]
-[process where process.parent.name == "vi" and process.name : ("bash", "sh")]
+[process where process.name : ("vi", "vim") and process.args : "-c" and process.args : (":!/bin/bash", ":!/bin/sh", ":!bash", ":!sh")]
```

for case sensitive multi-values check you can use `something in ("val1", "val2")` will match only on `val1` and `val2` and not on `vAl1` or `VAL1`

```suggestion
[process where process.name in ("vi", "vim") and process.args == "-c" and process.args in (":!/bin/bash", ":!/bin/sh", ":!bash", ":!sh")]
```